pkg/littlefs/Makefile.littlefs

```diff
@@ -1,6 +1,8 @@
 MODULE := littlefs
 
-CFLAGS += -Wno-unused-parameter -Wno-format
+CFLAGS += -Wno-format
+# TODO: remove when upstream is fixed
```

Comment:
Seems like this is fixed in `v1.6.2` :)

Reply:
You're more responsive than I am ^^. I will update tomorrow.